tox.ini

```
@@ -7,7 +7,7 @@ env_list = lint, integration
 setenv =
 	version=3.6.4
 	repo=https://github.com/canonical/zookeeper-k8s-operator.git
-	base-name=docker.io/dataplatformoci/zookeeper
+	base-name=ghcr.io/canonical/charmed-zookeeper:3.6.4-22.04_edge
```

I prefer to store the version only inside `rockcraft.yaml` without hardcoding here (`yq .version rockcraft.yaml` can help). see [comment below](https://github.com/canonical/zookeeper-rock/pull/15#discussion_r1155676048)
```suggestion
	base-name=ghcr.io/canonical/charmed-zookeeper
```